docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,178 @@ up OpenAI authentication to run the code below.
+This is particularly useful when you want to evaluate a deployed model hosted by a local `MLflow Deployments Server <../deployments/index.html>`_ or `Databricks Foundation Models API and External Models <https://docs.databricks.com/en/machine-learning/model-serving/score-foundation-models.html>`_, without implementing custom prediction logic to wrap it as an MLflow model or a python function.
+
+Please don't forget to set the target deployment client by using :py:func:`mlflow.deployments.set_deployments_target` before calling :py:func:`mlflow.evaluate()` with the endpoint URI, as shown in the example below. Otherwise, you will see an error message like ``MlflowException: No deployments target has been set...``.
+
+.. hint::
+
+    When you want to use an external endpoint **not** hosted by an MLflow Deployments Server or Databricks, you can create a custom Python function following the :ref:`Evaluating with a Custom Function <llm-eval-custom-function>` guide and use it as the ``model`` argument.
+
+Supported Input Data Formats
+****************************
+
+The input data can be either of the following format when using an URI of the MLflow Deployment Endpoint as the model:
+
+.. list-table::
+    :widths: 20 40 40
+    :header-rows: 1
+    :class: wrap-table
+
+    * - Data Format
+      - Example
+      - Additional Notes
+
+    * - A pandas DataFrame with a string column.
+      - 
+        .. code-block:: python
+
+            {
+                "inputs": [
+                    "What is MLflow?",
+                    "What is Spark?",
+                ]
+            }
+
+      - For this input format, MLflow will construct the appropriate request payload to the model endpoint type. For example, if your model is a chat endpoint (``llm/v1/chat``), MLflow will wrap your input string with the chat messages format like ``{"messages": [{"role": "user", "content": "What is MLflow?"}]}``. If you want to customize the request payload e.g. including system prompt, please use the next format.
+
+    * - A pandas DataFrame with a dictionary column.
+      - 
+        .. code-block:: python
+
+            {
+                "inputs": [
+                    {
+                        "messages": [
+                            {"role": "system", "content": "Please answer."},
+                            {"role": "user", "content": "What is MLflow?"},
+                        ],
+                        "max_tokens": 100,
+                    },
+                    # ... more dictionary records
+                ]
+            }
+
+      - In this format, the dictionary should have the correct request format for your model endpoint.
+
+    * - A (nested) list of input strings.
+      - 
+        .. code-block:: python
+
+            [
+                ["What is MLflow?"],
+                ["What is Spark?"],
+            ]
+
+      - The :py:func:`mlflow.evaluate()` will also accepts a list input. One notable requirement is that the each
```

Comment:
note: This is not new limitation but the current behavior of evaluate API. It's possible to support flat list, but not sure if it breaks other existing use cases, so just keep the behavior in this PR.

Reply:
Can we add that flat list support?
Hopefully existing CI will catch issues?